Source/SessionManager.swift

```diff
@@ -874,12 +874,16 @@ open class SessionManager {
                     return
                 }
 
-                let retrySucceeded = strongSelf.retry(request)
+                DispatchQueue.utility.after(timeDelay) {
```

Comment:
I'm unclear as to whether we should be executing this code on a separate queue or the same queue that originally executed the request. Wouldn't this logic create a new queue for every retry attempt? 

Perhaps at some point one of us should create a runtime diagram that outlines the various queues used...